CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ All notable changes to the Wazuh app project will be documented in this file.
 ### Fixed
 
 - Fixed path in logo customization section [#4352](https://github.com/wazuh/wazuh-kibana-app/pull/4352)
+- Fixed a TypeError in Firefox [#4362](https://github.com/wazuh/wazuh-kibana-app/pull/4362)
```

Comment:
This is too generic.

My suggestion is: 
> A compatibility problem with browsers using a different engine than V8 has been fixed. This problem affected Firefox and Chrome browsers.

Reply:
This comment was resolved with this commit [f944eeb](https://github.com/wazuh/wazuh-kibana-app/pull/4362/commits/f944eebb60244be1957eccefc17e665a412e19b1)